apps/devhub/tasks.py

```diff
@@ -144,6 +144,17 @@ def handle_file_validation_result(results, file_id, annotate=True):
     return FileValidation.from_json(file_, results)
 
 
+def addon_can_be_signed(validation):
+    """
+    Given a dict of add-on validation results, returns True if add-on can be
+    signed.
+    """
+    summary = validation.get('signing_summary', {})
+    return (summary.get('low', 0) +
```

Comment:
I know this is copy/paste but this may as well be `summary.get('low', 0) > 0 or ...` instead of adding and comparing to `0` right? Probably doesn't make a difference in the end but it seems silly to add the numbers together.